assets/css/style.css

```diff
@@ -309,3 +307,110 @@ hr {
   text-align: center;
   color: #6070ff;
 }
+
+@media (min-width: 768px) {
+  /* header */
+  header {
+    height: 73px;
+  }
+
+  header .container img {
+    display: none;
+  }
+
+  header .container ul {
+    display: flex;
+    gap: 20px;
+    color: #344563;
+  }
+
+  #hero {
+    background: url("../images/Header\ bg.svg") no-repeat #fff 100%;
+    border-bottom-left-radius: 160px;
+    background-position: center;
+    background-size: contain;
+    padding: 50px 140px;
+  }
+
+  /* card */
+  #work {
+    margin-top: 104px;
+  }
+
+  .work {
+    padding: 0 20px;
+  }
+
+  .card {
+    flex-direction: row;
+  }
+
+  .card .card-img {
+    width: 100%;
+    padding-right: 24px;
+  }
```

Comment:
- [ ] Great effort working on your `desktop version` but currently your work cards need some attention and I would like you to stick to the Figma design as much as possible, In the Figma design, the `2nd` and `4th` card has a kind of reverse in content whereby the image goes to the right side and other content is on the left side. Kindly implement this kind of reversed property in your cards too in order to have a perfect replica of the Figma design. 

Figma Design 👇 

![org card](https://github.com/Izebiafe/my-protfolio/assets/78634841/db4da507-dcc7-425f-9fb4-61da1031c8d1)